spec/controllers/admin/ticket_scannings_controller_spec.rb

```diff
@@ -4,7 +4,9 @@
   let(:admin) { create(:admin) }
   let(:conference) { create(:conference) }
   let(:user) { create(:user) }
-  let(:paid_ticket_purchase) { create(:ticket_purchase, conference: conference, user: user) }
+  let!(:registration) { create(:registration, conference: conference, user: user) }
```

Comment:
is this testing this? I would say we miss tests here :thinking:

Reply:
It needs tests, sure.

Reply:
done.

Reply:
@siddhantbajaj why are this new let exactly needed here?

Reply:
To test the create action of TicketScanning controller, user must registered for the conference. But yes ideally it should be handled in ticket_purchase factory because user must registered for the conference before purchasing its tickets. I'll open a different PR for modifying factories. 👍

Reply:
here would be better, but ok! :+1: